include/eeprom/core/types.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
-namespace eeprom_types {
+namespace eeprom {
+namespace types {
 
```

Comment:
is this the same as `namespace eeprom::types{}` ?

Reply:
Yup

Reply:
It is. I'll rename. Thanks